reactor-netty-http/src/main/java/reactor/netty/http/client/Http2ConnectionProvider.java

```diff
@@ -149,6 +150,11 @@ protected InstrumentedPool<Connection> createPool(
 		return new PooledConnectionAllocator(id, name(), parent, config, poolFactory, remoteAddress, resolverGroup).pool;
 	}
 
+	@Override
+	protected ConnectionPoolMetrics delegateConnectionPoolMetrics(InstrumentedPool.PoolMetrics metrics) {
+		return new Http2DelegatingConnectionPoolMetrics((Http2Pool) metrics);
+	}
```

Comment:
In the case of using the HTTP/2 protocol, provide `Http2DelegatingConnectionPoolMetrics` metrics.